app/assets/javascripts/components/action.js.jsx

```diff
@@ -1,7 +1,15 @@
 var Action = React.createClass({
   render: function () {
+    if (this.props.data.title === 'Ping') {
```

Comment:
Could the `Ping` class compose the `Action` component and not the other way around?

Reply:
Fixed!